dists/fs-repo-migrations/matrices/v1.0.0

```diff
@@ -0,0 +1,10 @@
+darwin 386
```

Comment:
Hold on, i think we do need `darwin 386`. I remember running into problems in pre-Yosemite with the mac mini and having to recompile as 386. 

If the go team has it there _still_, it's there for a reason. Let's keep it.